network/src/commonMain/kotlin/com/wire/kalium/network/NetworkClient.kt

```diff
@@ -110,15 +110,14 @@ internal fun provideBaseHttpClient(
 ) = HttpClient(engine) {
 
     if (NetworkLogger.isRequestLoggingEnabled) {
-        install(KaliumKtorCustomLogging) {
+        Logging {
```

Comment:
Unrelated change which should we reverted?

Reply:
oh! this one slipped out, sure will remove it. thanks